### PR TITLE
windowsdesktop-runtime: Update info

### PR DESCRIPTION
**Pull request by b4imetu**
Closes #XXXX

Relates to #XXXX

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/windowsdesktop-runtime.json [hunk 1]
```diff
@@ -1,9 +1,9 @@
 {
     "version": "8.0.0",
-    "description": "Microsoft .NET 7.0 Desktop Runtime",
+    "description": "Microsoft .NET 8.0 Desktop Runtime",
     "homepage": "https://dotnet.microsoft.com/download/dotnet",
     "license": "MIT",
-    "notes": "You can now remove this installer with 'scoop uninstall windowsdesktop-runtime'",
+    "notes": "You can now remove this installer with 'scoop uninstall -p windowsdesktop-runtime'",
```
Comment (HUMORCE) [inline bucket/windowsdesktop-runtime.json:6]:
```suggestion
    "notes": "You can now remove this installer with 'scoop uninstall windowsdesktop-runtime'",
```
No persistence data has been created.

**Review by HUMORCE — CHANGES_REQUESTED** [review]

**Comment by b4imetu** [comment]
https://github.com/ScoopInstaller/Versions/pull/1457
Please also take a look at this PR, as a non-latest but Long Term Support version supplement

**Comment by HUMORCE** [comment]
#12363

**Pull request closed without merging**